### PR TITLE
feat: Enrich notification message

### PR DESCRIPTION
**Pull request by kumada626**
## Context
Currently, notification message which is sent to slack has little information about a build.
I added commit information to notification message and refine message style.

## Objective
Fix message body and attachment fields.
New notification message like below.

### CODE REVIEW

**Comment by wahapo** [comment]
I want to see what it looks like. Is there a UI example?

**Comment by kumada626** [comment]
I added examples of messages in description.

File: index.js [hunk 1]
```diff
@@ -94,20 +101,18 @@ class SlackNotifier extends NotificationBase {
...
-                    {
-                        title: 'Build',
-                        value: `<${buildData.buildLink}|#${buildData.build.id}>`,
-                        short: true
-                    }
-                ]
+                title: `#${buildData.build.id}`,
+                title_link: `${buildData.buildLink}`,
+                // eslint-disable-next-line max-len
+                text: `${buildData.event.commit.message} (<${buildData.event.commit.url}|${truncatedSha}>)` +
```
Comment (jithin1987) [inline index.js]: You will have to truncate the commit message. We've seen cases where commit message overflows. 

That was the reason we did this fix for UI https://github.com/screwdriver-cd/ui/pull/399/files#diff-007d51a1584c112070731fd1fcdd9504R63

Comment (jithin1987) [inline index.js]: You will have to truncate the commit message. We've seen cases where commit message overflows. 

That was the reason we did this fix for UI https://github.com/screwdriver-cd/ui/pull/399/files#diff-007d51a1584c112070731fd1fcdd9504R63

Comment (kumada626) [reply index.js]: I fixed to truncate the commit message if commit message length over 150.

**Comment by tkyi** [comment]
I know originally the team was trying to make the Slack notification as minimal/small as possible so it didn't take up too much screen space. Would it make sense to have a `simple` or `minimized` option for Slack notifications?

**Comment by kumada626** [comment]
That's make sense to have a `simple` or `minimized` option for Slack notifications. I added `minimized` option. When this options is true, Slack notification message is the same as before.

**Pull request merged**